integration/src/test/java/org/hibernate/validator/integration/wildfly/jpa/CustomValidatorFactoryInPersistenceUnitIT.java

```diff
@@ -86,6 +89,31 @@ public void testValidatorFactoryPassedToPersistenceUnitIsCorrectlyConfigured() t
 		log.debug( "testValidatorFactoryPassedToPersistenceUnitIsCorrectlyConfigured completed" );
 	}
 
+	/**
+	 * Makes sure the HV added via our module ZIP is used instead the one coming with WF by relying on functionality not
+	 * present in the WF-provided HV
+	 */
+	// TODO How to make that work reliably also after a HV upgrade within WF?
```

Comment:
Yes, I hope we can find a better way. Maybe checking on the protection domain of the HV JAR or so.